BDArmory/Modules/ModuleWeapon.cs

```diff
@@ -3135,7 +3135,7 @@ public void RunTrajectorySimulation()
                     if (eWeaponType == WeaponTypes.Ballistic && bulletDrop)
                     {
                         var timeOfFlight = BallisticTrajectorySimulation(ref simCurrPos, simVelocity, Mathf.Min(maxTargetingRange, targetDistance), maxTargetingRange / bulletVelocity, simDeltaTime, FlightGlobals.getAltitudeAtPos(targetPosition) < 0);
-                        bulletPrediction = simCurrPos;
+                        //bulletPrediction = simCurrPos;
```

Comment:
@SuicidalInsanity, looking at this in regards to issue #297 (which has the same problem), I don't see how these changes actually do anything.
`simCurrPos` is passed as a reference, so setting `bulletPrediction = simCurrPos;` after `BallisticTrajectorySimulation` returns should be the same as setting it in the various places below.